crystals/lattice.py

```diff
@@ -174,6 +192,11 @@ def periodicity(self) -> Tuple[float, float, float]:
         # absolutes of columns
         lv = np.abs(np.array(self.lattice_vectors))
         return tuple(lv.sum(axis=0))
+    
+    def gLength(self, ghkl) -> float:
```

Comment:
What is this supposed to be? Can it not be derived from other methods, such as `scattering_vector`?